src/components/EventRegistrantsModal/Modal/AddOnSpot/AddOnSpotAttendee.tsx

```
@@ -0,0 +1,230 @@
+/**
+ * Component for adding an attendee on the spot via a modal form.
+ *
+ * This component provides a modal interface to add attendees to an event
+ * by collecting their details such as name, email, phone number, and gender.
+ * It validates the form inputs, submits the data to the server using a GraphQL
+ * mutation, and handles success or error responses appropriately.
+ *
+ * @component
```

ESLint

[error] tsdoc-undefined-tag: The TSDoc tag "@component" is not defined in this configuration

(tsdoc/syntax)